oracle/types/market.go

```diff
@@ -110,8 +113,8 @@ func (pmm *ProviderMarketMap) ValidateBasic() error {
 	if len(pmm.Name) == 0 {
 		return fmt.Errorf("provider name cannot be empty")
 	}
-	if len(pmm.OffChainMap) != len(pmm.TickerConfigs) {
-		return fmt.Errorf("off-chain map length mismatch")
+	if len(pmm.OffChainMap) > len(pmm.TickerConfigs) {
+		return fmt.Errorf("off-chain map length invalid %d>%d, %v %v", len(pmm.OffChainMap), len(pmm.TickerConfigs), pmm.OffChainMap, pmm.TickerConfigs)
```

Comment:
Why would the lengths be different?

Reply:
The wonky behavior of duplicate feeds (this goes away with our proposed fix